### PR TITLE
Append slash to urls

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-2692)

append-slash

### CODE REVIEW

File: ui/urls.py [hunk 1]
```diff
@@ -60,81 +60,81 @@
...
     ),
     url(
-        r'^suppliers$',
+        r'^suppliers/$',
         PublishedProfileListView.as_view(),
         name='public-company-profiles-list',
     ),
     url(
-        r'^search$',
+        r'^search/$',
```
Comment (sdonk) [inline ui/urls.py]: Can we use `APPEND_SLASH`?

https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-APPEND_SLASH

Comment (konradko) [reply ui/urls.py]: We need to, but that works if you have slashes in URLs and user requests without a slash. Also does not work for POST, so we need to have slashes in api client urls

Comment (konradko) [reply ui/urls.py]: Ah, actually it’s True by default - we should still add to be explicit

Comment (richtier) [reply ui/urls.py]: `APPEND_SLASH` is indeed `True` by default, and `CommonMiddleware` does the redirecting.

Let's not de-defined default settings as that will make the settings file noisy.

**Pull request merged**